src/main/kotlin/frc/robot/commandGroups/CommandGroups.kt

```diff
@@ -82,4 +80,43 @@ object CommandGroups {
             (gripper.setRollerPower(-0.7))
         ).withName("outtake")
     }
+
+    fun superPoopInit(): Runnable {
```

Comment:
Nobody can understand WTH that means.

Reply:
5e052820aa3ec43699968b8f1e1894eb38d75eb3